doc/WebRTC.xml

```diff
@@ -406,14 +396,14 @@
         <varlistentry>
           <term>request</term>
           <listitem>
-              <para role="param">peer optional [string]</para>
-            <para role="text">The ID of the peer to connect to.</para>
+            <para role="param">peer optional [string]</para>
+            <para role="text">The ID of the peer to connect to. This Id is defined outside of this specification and must be negotiated out-of-band.</para>
             <para role="param">authorization [JWT]</para>
             <para role="text">Access token that authorizes the device or client.</para>
             <para role="param">session optional [string]</para>
             <para role="text">The ID assigned by the signaling server to the session.</para>
             <para role="param">profile optional [string]</para>
-              <para role="text">Optional token of the media streaming profile to use.</para>

```

Comment:
Removing the **optional** here contradicts with the mandatory 'default profile' parameter in WebRTCConfiguration
Ref: view-source:https://www.onvif.org/ver20/media/wsdl/media.wsdl

```xml
<xs:element name="DefaultProfile" type="tt:ReferenceToken">
	<xs:annotation>
		<xs:documentation>The default media profile to use for streaming if no specific profile is specified when initializing a session.</xs:documentation>
	</xs:annotation>
</xs:element>
```

Reply:
We simply removed the redundant "optional" word in the description, because the field itself is already marked as optional as seen on line 405.